src/ol/control/overviewmapcontrol.js

```diff
@@ -202,7 +202,7 @@ ol.control.OverviewMap.prototype.bindView_ = function() {
   // FIXME - the overviewmap view rotation currently follows the one used
   // by the main map view.  We could support box rotation instead.  The choice
   // between the 2 modes would be made in a single option
-  this.ovmap_.getView().bindTo(ol.ViewProperty.ROTATION, view);
+  // this.ovmap_.getView().bindTo(ol.ViewProperty.ROTATION, view);
```

Comment:
This can be replaced with a listener for rotation changes.  The listener should be unregistered when the map's view changes (which is not being done currently).